src/pages/resource/ResourcePage.tsx

```diff
@@ -55,11 +77,28 @@ const ResourcePage: FC<RouteProps> = (props) => {
     }
   }, [identifier]);
 
-  return (
+  const isAuthor = () => resource.features.dlr_submitter_email === user.email;
```

Comment:
hva synes du om å trekke ut på denne måten for lesbarhet ?

Reply:
Synes det ser greit ut.